library/os.path.po

```diff
@@ -524,22 +607,28 @@ msgid ""
 "specifications, *drive* will always be the empty string.  In all cases, "
 "``drive + tail`` will be the same as *path*."
 msgstr ""
+"將路徑名 *path* 拆分為 ``(drive, tail)`` 一對，其中 *drive* 是掛載點或空字"
+"串。在不使用驅動機規範的系統上，*drive* 將始終為空字串。在所有情況下，"
+"``drive + tail`` 將與 *path* 相同。"
 
 #: ../../library/os.path.rst:491
 msgid ""
 "On Windows, splits a pathname into drive/UNC sharepoint and relative path."
-msgstr ""
+msgstr "在 Windows 上，將路徑名拆分為驅動機或 UNC 共享點以及相對路徑。"
 
 #: ../../library/os.path.rst:493
 msgid ""
 "If the path contains a drive letter, drive will contain everything up to and "
 "including the colon::"
 msgstr ""
+"如果路徑包含驅動機字母，則 *drive* 將包含從頭到冒號（包括冒號）的所有內容："
 
 #: ../../library/os.path.rst:499
 msgid ""
 "If the path contains a UNC path, drive will contain the host name and share::"
-msgstr ""
+msgstr "如果路徑包含 UNC 路徑，則驅動機將包含主機名和共享名：\\n"
```

Comment:
My bad. The first time suggestion I unexpectedly typed "\\n", the correct one is "\n".

```suggestion
msgstr "如果路徑包含 UNC 路徑，則驅動機將包含主機名和共享名：\n"
```